napari/_tests/test_multiple_viewers.py

```diff
@@ -1,7 +1,10 @@
+import pytest
+
 from napari import Viewer
 
 
-def test_multi_viewers_dont_clash(qapp):
+@pytest.mark.skip(reason="problem with clean")
+def test_multi_viewers_dont_clash(qtbot):
```

Comment:
Did this test start causing problems after the changes here, or is it independently flaky?

Reply:
This is the only test in which we use `Viewer` instead of `make_napari_viewer` so it does not go through this check:
https://github.com/napari/napari/blob/f6bdd6235da23d4d23239653b6e4cf02e7f6d4ac/napari/utils/_testsupport.py#L233-L236 

 So there is only one place where viewers may leak.